HelloWeb/README.md

```diff
@@ -0,0 +1,27 @@
+# HelloWeb Application
+
+Gluon Application on the browser
+
+## Basic Requirements
+
+A list of the basic requirements can be found online in the [Gluon documentation](https://docs.gluonhq.com/#_requirements).
+
+## Quick instructions
+
+### Run the sample
+
+    mvn javafx:run
+
+### Run the sample in browser
+
+    mvn gluonfx:build gluonfx:nativerun -Pweb
```

Comment:
Do we need to mention that Chrome should be default browser? Else maybe:
```
mvn gluonfx:build -Pweb

```
and then open `target/gluonfx/x86_64-web/gvm/web/index.html` with Chrome?